buildprocess/webpack-docs.config.js

```diff
@@ -0,0 +1,41 @@
+var configureWebpackForTerriaJS = require("./configureWebpack");
+var MiniCssExtractPlugin = require("mini-css-extract-plugin");
+var path = require("path");
+
+module.exports = function() {
+  const config = {
+    mode: "development",
+    entry: path.resolve(__dirname, "generateDocs.js"),
+    output: {
+      path: path.resolve(__dirname, "..", "build"),
+      filename: "generateDocs.js",
+      sourcePrefix: "", // to avoid breaking multi-line string literals by inserting extra tabs.
+      globalObject: "(self || window)" // to avoid breaking in web worker (https://github.com/webpack/webpack/issues/6642)
+    },
+    target: "node",
+    resolve: {
+      alias: {},
+      modules: ["node_modules"],
+      extensions: [".ts", ".js", ".jsx", ".tsx", ".json"]
+    },
+    externals: {
+      vue: "vue"
```

Comment:
Is this being used?

Reply:
No

Reply:
Turns out it was 😂 . I've re-added it. Documentation.js uses the template renderer from vuejs and it tries to detect what Vue version is in use (or something). These couple of lines fix webpack wanting vuejs to be a dependency.